spec/active_triples/rdf_source_spec.rb

```diff
@@ -720,4 +720,22 @@ class MyDataModel < ActiveTriples::Schema
       expect { dummy_source.new.test_title }.not_to raise_error
     end
   end
+
+  describe 'callback' do
+    let(:callback) {double}
+
+    before do
+      class TestRdfSource
```

Comment:
Can you remove this class in an `after` block, or create an anonymous class (e.g. `Class.new do ... end`)